src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/model/ManagedIndexMetaData.kt

```diff
@@ -65,6 +67,23 @@ data class ManagedIndexMetaData(
         )
     }
 
+    fun asTemplateArg(): Map<String, Any?> {
```

Comment:
Can we reuse toXContent output to give back to customer so that we have same log to remove unnecessary information back to users? I know toXContent is not a Map but since it is a `XContentBuilder` should be easy to transform it back to a Map

Reply:
Discussed offline and we will utilize the toXContent unless we see requests for full ManagedIndexMetaData in ctx variable.